operations/app/src/modules/database/main.tf

```diff
@@ -148,6 +148,18 @@ resource "azurerm_postgresql_database" "prime_data_hub_db" {
   }
 }
 
+resource "azurerm_postgresql_database" "prime_data_hub_candidate_db" {
+  name = "prime_data_hub_candidate"
+  resource_group_name = var.resource_group
+  server_name = azurerm_postgresql_server.postgres_server.name
+  charset = "UTF8"
+  collation = "English_United States.1252"
```

Comment:
I think this collation is ANSI Latin 1 and not UTF-8. Since the CharSet is listed as `UTF-8`, is this intentional? Did you mean `en_US.UTF-8`? The settings above may give unexpected sorting results IIRC.
I do note that other DBs have been created similarly as listed above.

Reply:
I checked our prod database and `English_United States.1252` is how it's currently configured. I don't disagree on the sorting issue, but thinking we should address separately, since it would involve migrate data.

Reply:
Concur